packages/search/lib/dropElasticsearch.js

```diff
@@ -1,9 +1,13 @@
+let debug = console.log
 const Elasticsearch = require('@orbiting/backend-modules-base/lib/Elasticsearch')
 
-const drop = async (prefix) => {
+const drop = async (prefix, doDebug) => {
   if (!prefix) {
     throw new Error("can't drop, prefix not specified")
   }
+  if (doDebug !== undefined && !doDebug) {
```

Comment:
```suggestion
const drop = async (prefix, doDebug = true) => {
  if (!prefix) {
    throw new Error("can't drop, prefix not specified")
  }
  if (!doDebug) {
```

Gives me less to think ;)